### PR TITLE
feat: Don't save `page` metadata anymore

### PR DESCRIPTION
**Pull request by Merkur39**
For the feature: Create a single document for double-sided papers.

The `page` attribute doesn't have to disappear completely, it's still useful during creation (e.g. AA crash recovery).
It's just no longer relevant to save it in the couch document.

As for the rest, the commits should be clear enough.

```
### ✨ Features

* Create a single document for double-sided papers.

### 🔧 Tech

* Refactor forward process
```

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -28,3 +28,12 @@ export const ANDROID_APP_URL =
...
 export const FLAGSHIP_SCAN_TEMP_FILENAME = 'flagshipScanTemp.png'
+
+export const TWO_SIDED_FILE_QUALIFICATION_LABELS = [
+  'national_id_card',
+  'driver_license',
+  'residence_permit',
+  'transport_card',
+  'fidelity_card',
+  'library_card'
+]
```
Comment (zatteo) [inline src/constants.js:39]: No possibility to derive this information from papersDefinition ?

**Comment by zatteo** [comment]
Do you still see the "Add" and "Forward" fab buttons on desktop ? 
Screenshot 2024-06-19 at 11 43 10

**Comment by zatteo** [comment]
Very nice refactoring of the "Share" and "Forward" mess 👍

**Review by zatteo — APPROVED** [review]

**Comment by Merkur39** [comment]
@zatteo Thanks for review, following our exchange I've added 3 last commits 👍

**Review by zatteo — APPROVED** [review]

**Pull request merged**